guides/routing.md

```diff
@@ -87,6 +87,36 @@ new Router()
   .use(nextRoutes)
 ```
 
+### Altering requests and responses
+
+XDN offers APIs to manipulate request and response headers and cookies. The APIs are:
+
+| Operation     | Request                 | Upstream Response              | XDN Response           |
+|---------------|-------------------------|--------------------------------|------------------------|
+| Set header    | `setRequestHeader`      | `setUpstreamResponseHeader`    | `setResponseHeader`    |
+| Add cookie    | `*`                     | `addUpstreamResponseCookie`    | `addResponseCookie`    |
+| Update header | `updateRequestHeader`   | `updateUpstreamResponseHeader` | `updateResponseHeader` |
+| Update cookie | `*`                     | `updateUpstreamResponseCookie` | `updateResponseCookie` |
+| Remove header | `removeRequestHeader`   | `removeUpstreamResponseHeader` | `removeResponseHeader` |
+| Remove cookie | `*`                     | `removeUpstreamResponseCookie` | `removeResponseCookie` |
+
+`*` Adding, updating and removing a request cookie can all be achieved with `updateRequestHeader` applied to `cookie` header.
```

Comment:
```suggestion
`*` Adding, updating, or removing a request cookie can all be achieved with `updateRequestHeader` applied to `cookie` header.
```

Reply:
For this one we actually don't have specific APIs so it is either "all" or "" but not "also" (no alternative but those)

Reply:
Ah, I misread.  Updated the suggestion.